src/include/processor/operator/persistent/reader/csv/driver.h

```diff
@@ -16,6 +16,8 @@ namespace processor {
 class BaseCSVReader;
 class ParsingDriver {
     common::DataChunk& chunk;
+
+protected:
     bool rowEmpty;
```

Comment:
Can u move these two fields below functions? Our code convention is class member fields are below functions. For example https://github.com/kuzudb/kuzu/blob/master/src/include/storage/storage_manager.h.

Same for other driver class in this file.

Reply:
Thanks for pointing this out, let me fix it.